src/editor.js

```diff
@@ -101,6 +101,8 @@ export class OdooEditor extends EventTarget {
 
         this.document = options.document || document;
 
+        this.isMobile = matchMedia('(max-width: 767px)').matches;
```

Comment:
I guess it's useless to listen to the changes of the media query list returned by match media?

Reply:
You mean, to support resizing of the page? Odoo doesn't support that, you have to reload the page. So I think that's not needed for now.